django/db/models/fields/related.py

```diff
@@ -289,9 +289,15 @@ def contribute_to_class(self, cls, name, virtual_only=False):
         self.opts = cls._meta
 
         if not cls._meta.abstract:
+
             if self.remote_field.related_name:
-                related_name = force_text(self.remote_field.related_name) % {
+                related_name = self.remote_field.related_name
+            else:
+                related_name = self.opts.default_related_name
+            if related_name:
+                related_name = force_text(related_name) % {
                     'class': cls.__name__.lower(),
+                    'model_name': cls._meta.model_name.lower(),
```

Comment:
Remove this line as `cls._meta.model_name.lower() == cls.__name__.lower()` and we already document using `class`.